components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/listener/SCIMUserOperationListener.java

```diff
@@ -539,4 +542,16 @@ public boolean doPostGetUserClaimValuesWithID(String userID, String[] claims, St
         claimMap.put(SCIMCommonConstants.READ_ONLY_USER_CLAIM, String.valueOf(userStoreManager.isReadOnly()));
         return true;
     }
+
+    private boolean isIdentityClaimUpdate(Map<String, String> claims) {
+
+        boolean isIdentityClaimUpdate = false;
+        for (Map.Entry<String, String> claim : claims.entrySet()) {
+            if (claim.getKey().contains(UserCoreConstants.ClaimTypeURIs.IDENTITY_CLAIM_URI)) {
```

Comment:
We need to check `contains` or `startsWith`?

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/376/commits/885367b448d53ffdb5414c784b8fccf532677ebb